src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -432,6 +450,57 @@ bval example:
 and thus define those values for all sessions and/or subjects in one place (see
 Inheritance principle).
 
+As an exception to the [common principle](../02-common-principles.md#definitions)
+that parameters are constant across runs, the gradient table information (stored
+within the `.bval` and `.bvec` files) MAY change across DWI runs.
+
+### Multipart (split) DWI schemes
+
+Some MR schemes cannot be acquired directly by some scanner devices,
+requiring to generate several DWI runs that were originally meant to belong
+in a single one.
+For instance, some GE scanners cannot collect more than &asymp;160 volumes
+in a single run under fast-changing gradients, so acquiring *HCP-style*
+diffusion images will require splitting the DWI scheme in several runs.
+
+| **Key name**    | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                      |
+| --------------- | --------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| MultipartID     | REQUIRED              | [string][]    | Unique identifier (participant's level) to prescribe DWI segments belonging in a single dataset. Segments can be designated by any combination of `dir-<label>`, `acq-<label>` and/or `run-<index>` key/value pairs. |
+
+JSON example:
+
+```JSON
+{
+  "PhaseEncodingDirection": "j-",
+  "TotalReadoutTime": 0.095,
+  "MultipartID": "dwi_protocol1"
+}
+```
+
+For instance, the above JSON sidecar defined for a set of files via the
+`sub-<label>_dwi.json` file would indicate that all the runs 1 through 4
+belong in the same unique image:
+
+```Text
+sub-<label>/[ses-<label>/]
+    dwi/
+       sub-<label>_run-1_dwi.nii.gz
+       sub-<label>_run-1_dwi.bval
+       sub-<label>_run-1_dwi.bvec
+       sub-<label>_run-2_dwi.nii.gz
+       sub-<label>_run-2_dwi.bval
+       sub-<label>_run-2_dwi.bvec
+       sub-<label>_run-3_dwi.nii.gz
+       sub-<label>_run-3_dwi.bval
+       sub-<label>_run-3_dwi.bvec
+       sub-<label>_run-4_dwi.nii.gz
+       sub-<label>_run-4_dwi.bval
+       sub-<label>_run-4_dwi.bvec
+       sub-<label>_dwi.json
```

Comment:
Would a json file for each of the individual runs no longer exist? Or would `sub-<label>_dwi.json` contain the unique tags that would trickle down and supersede the tags in the individual run jsons?

Reply:
This is just to make the example shorter, but the rationale of this new tag `MultipartID` still works with multiple jsons.